internal/engine/podlogmanager_test.go

```diff
@@ -271,6 +310,7 @@ func (f *plmFixture) AssertOutputContains(s string) {
 }
 
 func (f *plmFixture) AssertOutputDoesNotContain(s string) {
+	time.Sleep(10 * time.Millisecond)
```

Comment:
👀 is this necessary?

Reply:
ya, otherwise we risk calling this before there is any output and it'll always return true, i.e. it's a useless assertion. (case in point: without this sleep, I couldn't make this test go red/green.)